setup/github-bot/README.md

```diff
@@ -0,0 +1,25 @@
+# node.js Build github-bot Setup
+
+For setting up a Debian 8 box for github-bot
+
+```text
+Host infra-rackspace-debian8-x64-2
+  HostName 119.9.53.51
+  User root
+```
+
+Note that these hostnames are also used in the *ansible-inventory* file. The IP addresses will need to be updated each time the servers are reprovisioned.
+
+To set up a host, run:
+
+```text
+$ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml
+```
+
+Start-up of the github-bot service:
+
+```text
+$ systemctl start github-bot
+```
+
+**Users**: The ansible-vars.yaml file contains a list of users who's GitHub public keys are pulled and placed into authorized_keys for both root and iojs users. This file should be updates when new users are added to the build project who are able to help maintain the containerized builds.
```

Comment:
This is old. I suggest you use a newer `README.md` template such as this: https://github.com/jbergstroem/build/blob/65e5ac8e7dbf8a35dce3a78c0bba679e2d295a7f/setup/fedora/README.md